JS/Yeva/health-app/README.md

```diff
@@ -0,0 +1,5 @@
+This example was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
@Yeva9 jan please never commit the code/file which is not yours. We ask you to copy the code which you need but do not absolutely copy everything. This file links to original source you need to delete it or make it your own